src/revocation.rs

```diff
@@ -8,9 +8,177 @@ use crate::error::{Error, Result};
 use crate::secure_mount;
 
 use std::path::Path;
+use std::process::{Child, Command, Output, Stdio};
 
 use serde_json::Value;
 
+// TODO: Add more types
+#[derive(Debug, Copy, Clone)]
+pub(crate) enum CmdType {
```

Comment:
If we expect the command to be self contained per the last sync (IE: it can be directly executed) why differentiate?

Reply:
You're right, this was over-engineered. Thanks!